lib/processCss.js

```diff
@@ -158,18 +161,18 @@ module.exports = function processCss(inputSource, inputMap, options, callback) {
 	var pipeline = postcss([
 		localByDefault({
 			mode: options.mode,
-			rewriteUrl: function(global, url) {
+			rewriteUrl: function(global, origUrl) {
 				if(parserOptions.url){
-					url = url.trim(" ");
+					var url = origUrl.trim(" ");
 
-					if(!loaderUtils.isUrlRequest(url, root)) {
-						return url;
+					if(!origUrl.replace(/\s/g, '').length || !loaderUtils.isUrlRequest(url, root)) {
+						return origUrl;
```

Comment:
@michael-ciniawsky about `url` i think best solution if `url` empty (also with spaces and newlines and etc) do nothing at all, it is good practice don't change css ast, if we do not need it, but here in fact we do not need it. The same applies to `import`. We can never know why the user did this, but if he did it maybe he really needed it, and by changing something we can break his logic. If there are a lot of issues with this, then we can just throw `warning`, why warning? because it is not fatal, all works as expected :smile: